src/app/core/search/search.service.ts

```diff
@@ -56,13 +56,15 @@ export class SearchService {
       const searchParameters = []
       if (escapedParams.firstName) {
         let searchValue = `given-names:${escapedParams.firstName}`
+        searchValue += ` OR given-and-family-names:${escapedParams.firstName}`
         if (escapedParams.otherFields === 'true') {
```

Comment:
@amontenegro @dandineen 

So if we are using `given-names:X OR given-and-family-names:` for given names, and `family-name:X given-and-family-names:X` for family names. 

Isn't it more simple to just use `given-and-family-names` for both name and family name inputs?

Reply:
As you have proposed on slack please use the given-and-family-names for inputs with diacritics

Reply:
Done!